cura/Settings/ExtruderManager.py

```diff
@@ -265,7 +265,8 @@ def getUsedExtruderStacks(self) -> List["ContainerStack"]:
 
         # The platform adhesion extruder. Not used if using none.
         if (global_stack.getProperty("adhesion_type", "value") != "none" or
-            global_stack.getProperty("prime_tower_brim_enable", "value")):
+            global_stack.getProperty("prime_tower_brim_enable", "value") and
```

Comment:
Shouldn't it be adhesion_type != None or (prime_tower_brim_enable && adhesion_type != raft) ?

Reply:
Yes.